main.go

```diff
@@ -174,15 +174,14 @@ func run(state overseer.State) {
 		engine.WithDetectors(!*noVerification, engine.DefaultDetectors()...),
 	)
 
-	filter, err := common.FilterFromFiles(*gitScanIncludePaths, *gitScanExcludePaths)
-	if err != nil {
-		logrus.WithError(err).Fatal("could not create filter")
-	}
-
 	var repoPath string
 	var remote bool
```

Comment:
nit: If we declare `var err error` here, within the switch we can use regular assignment `=` vs `:=`